### DISCUSSION TITLE
Trouble Authorizing Single Sign On Authorization

### DISCUSSION THREAD

**Discussion by eaa1310** [community/community#22786] [New to GitHub]
<p>I’m just getting an account set up, and I went through most of the steps to authorize single sign on.</p>
<p>I generated the SSH key, and added it to my account. When I get to the next step, it says there should be a button to Enable SSO, but there isn’t.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/697f9a4f8a45d4aa352b4c0c149226599dc9bcb6.png" data-download-href="/uploads/short-url/f3hwmTiIK0KqQJ7HUzwKCmLBYlE.png?dl=1" title="ssh.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/47125340/181073130-5cd7ab94-5a7e-46f1-921b-93f876412de3.png" alt="ssh.png" data-base62-sha1="f3hwmTiIK0KqQJ7HUzwKCmLBYlE" width="690" height="237" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/697f9a4f8a45d4aa352b4c0c149226599dc9bcb6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">ssh.png</span><span class="informations">766×264 13.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Does it look like I missed a step somewhere?</p>
<p>Thanks,</p>
<p>Paul</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/eaa1310">@eaa1310</a>,</p>
<p>Out of curiosity, are you part of an organization on GitHub that has SSO enabled?</p>
<p>Thanks!</p>

**Comment by eaa1310** [comment]
<p>No, at this time I’m just trying to use balena-cloud</p>
<p>which is a web application to remotely control digital signs. It uses GitHub’s SSO, so I was going through the steps to link a login here with that.</p>
<p>From your question it sound like I may have been going at this from the wrong direction.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/eaa1310">@eaa1310</a>,</p>
<p>Sorry for the delay, but yes, you are correct. The options to enable SSO for an SSH key related to the SSO for Organizations feature and would not impact external applications.</p>
<p>Hope this helps!</p>

**Comment by illia-sh** [comment, marked as answer]
<p>Hi, I had the same issue, just follow this link <a href="https://github.com/" rel="noopener nofollow ugc">https://github.com/</a>&lt; your-github-organization &gt;</p>
<p>On the top screen, you will be prompted to auth against your organization, Once you do this, the button with ENABLE SSO will appear.</p>
